workflows/quickstart/index.js

```diff
@@ -50,15 +50,16 @@ async function executeWorkflow(projectId, location, workflow) {
       setTimeout(resolve, ms);
     });
   }
-
+  // Runtime arguments can be passed as a JSON string
+  const runtimeArgs = searchTerm
+    ? JSON.stringify({searchTerm: searchTerm})
+    : '{}';
```

Comment:
Instead of hard-coding the object to a string, cleaner if you keep the original format `const runtimeArgs = searchTerm ? {searchTerm: searchTerm} : {};` and allow `JSON.stringify` do that work for you.